FV3/atmos_cubed_sphere/tools/coarse_grained_diagnostics.F90

```diff
@@ -586,6 +586,33 @@ subroutine populate_coarse_diag_type(Atm, coarse_diagnostics)
     coarse_diagnostics(index)%units = 'kg/m**2'
     coarse_diagnostics(index)%reduction_method = AREA_WEIGHTED
     coarse_diagnostics(index)%special_case = "total_ice_water_path"
+
+    index = index + 1
+    coarse_diagnostics(index)%axes = 2
+    coarse_diagnostics(index)%module_name = DYNAMICS
+    coarse_diagnostics(index)%name = 'tb_coarse'
```

Comment:
Why `tb_coarse` instead of `ts_coarse`?

Reply:
Indeed it breaks from the pattern used for the winds, but in this case `"tb"` is the name of the corresponding fine-resolution diagnostic: https://github.com/ai2cm/fv3gfs-fortran/blob/3238351a062385baae73f91bc121dcd77347b442/FV3/atmos_cubed_sphere/tools/fv_diagnostics.F90#L825-L826
In general I try to name the coarse diagnostics following the pattern `"{name_of_fine_diagnostic}_coarse"`.